Indent rule VariableDeclarator doesn't apply to arrow functions

Using ESLint `1.3.1` on this code
```
const someFunction = function (argument) {
        console.log(argument);
    },
    someValue = 'someValue';

const someOtherFunction = argument => {
        console.log(argument);
    },
    someOtherValue = 'someOtherValue';

export {
    someFunction,
    someOtherFunction,
    someOtherValue,
    someValue
};
```

with this .eslintrc
```
{
    "ecmaFeatures": {
        "arrowFunctions": true,
        "blockBindings": true,
        "modules": true
    },
    "root": true,
    "rules": {
        "indent": [
            2,
            4,
            {
                "SwitchCase": 1,
                "VariableDeclarator": 1
            }
        ]
    }
}
```

outputs
```
  7:9  error  Expected indentation of 4 space characters but found 8  indent
  8:5  error  Expected indentation of 0 space characters but found 4  indent

✖ 2 problems (2 errors, 0 warnings)
```

I expected regular functions and arrow functions to have the same indentation rules.

--- a/lib/rules/indent.js
+++ b/lib/rules/indent.js
@@ -416,7 +416,11 @@ module.exports = function(context) {
             return;
         }
 
-        if (node.parent && (node.parent.type === "FunctionExpression" || node.parent.type === "FunctionDeclaration")) {
+        if (node.parent && (
+                node.parent.type === "FunctionExpression" ||
+                node.parent.type === "FunctionDeclaration" ||
+                node.parent.type === "ArrowFunctionExpression"
+        )) {
             checkIndentInFunctionBlock(node);
             return;
         }

--- a/tests/lib/rules/indent.js
+++ b/tests/lib/rules/indent.js
@@ -810,6 +810,14 @@ ruleTester.run("indent", rule, {
                 "  }\n" +
                 "};\n",
             options: [2]
+        },
+        {
+            code:
+                "const someOtherFunction = argument => {\n" +
+                "        console.log(argument);\n" +
+                "    },\n" +
+                "    someOtherValue = 'someOtherValue';\n",
+            ecmaFeatures: { arrowFunctions: true, blockBindings: true }
         }
     ],
     invalid: [

